### PR TITLE
Update readme.md with usage and screenshot

### PR DESCRIPTION
**Pull request by ThatOneCalculator**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+2. Open Terminal > Settings > Gear icon at the bottom left > Import Profile
+3. Import the profile > Set as default
+
+If the file can't be selected:
+
+4. Export a random profile from the list with selecting it > gear icon > Export Profile
+5. Replace the contents of the file you just exported with that of rose-pine.terminal
+6. Import the profile > Set as default
+
+### Screenshots
```
Comment (mvllow) [inline readme.md]: let's change to h2

File: readme.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
 
 # Rosé Pine for Terminal.app
 
+> Maintained by our lovely community ☺️
+
 All natural pine, faux fur and a bit of soho vibes for the classy minimalist.
 
 [![](https://img.shields.io/badge/Rosé%20Pine%20Theme-191724)](https://github.com/rose-pine/rose-pine-theme)
+
+### Usage
```
Comment (mvllow) [inline readme.md]: also change to h2

File: readme.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
...
+
+4. Export a random profile from the list with selecting it > gear icon > Export Profile
+5. Replace the contents of the file you just exported with that of rose-pine.terminal
+6. Import the profile > Set as default
+
+### Screenshots
+
+Fetch featured below is [NerdFetch](https://github.com/thatonecalculator/nerdfetch)
+
+![preivew](https://i.ibb.co/BNrJbMH/Screen-Shot-2020-11-12-at-12-07-23-AM.png)
```
Comment (mvllow) [inline readme.md]: change preivew to preview

**Review by mvllow — CHANGES_REQUESTED** [review]

**Comment by ThatOneCalculator** [comment]
Sorry about that! Didn't see it till now. Made those changes!

**Review by mvllow — APPROVED** [review]
thanks!

**Pull request merged**